archimedes/archimedes/functions/dataflow/path.py

```diff
@@ -0,0 +1,27 @@
+# Do not let these imports become usable within scripts themselves
+import os as _os
+import os.path as _os_path
+
+
+def input_path(name, inputs_env=_os.environ, inputs_dir=None):
```

Comment:
These are just the lowest level primitives.  When we get to supporting pandas, we can have smarting `input_dataframes()` functions, etc, that look nicer.